src/sqlancer/mysql/MySQLProvider.java

```diff
@@ -148,7 +140,7 @@ private static int mapActions(MySQLGlobalState globalState, Action a) {
 
     @Override
     public void generateDatabase(MySQLGlobalState globalState) throws Exception {
-        while (globalState.getSchema().getDatabaseTables().size() < Randomly.smallNumber() + 1) {
+        while (globalState.getSchema().getDatabaseTables().size() < Randomly.getNotCachedInteger(1, 2)) {
```

Comment:
I think we could just leave `Randomly.smallNumber() + 1`, which might sometimes generate more tables than 2?

Reply:
No. Small number returns (-1, 1) with a Gaussian distribution, which means in most situations, there is only one table.

Reply:
Thanks for the clarification!